src/Symfony/Component/HttpClient/Response/ResponseTrait.php

```diff
@@ -244,6 +244,14 @@ private static function addResponseHeaders(array $responseHeaders, array &$info,
         }
     }
 
+    private static function handleContentType(&$content, string $contentType, string $bufferRegex): void
+    {
+        $contentType = substr($contentType, 0, strpos($contentType, ';'));
```

Comment:
the semi-colon is optional: strpos can return false

Reply:
Added an extra check to see if strpos returns false. In that case, I'll use the length of the contentType variable